java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -970,6 +970,28 @@ public ColumnVector normalizeNANsAndZeros() {
     return new ColumnVector(normalizeNANsAndZeros(getNativeView()));
   }
 
+  /**
+   * Create a deep copy of the column while replacing the null mask. The resultant null mask is the
+   * bitwise merge of null masks in the columns given as arguments.
+   *
+   * @param mergeOp binary operator, currently only BITWISE_AND is supported.
+   * @param columns array of columns whose null masks are merged, must have identical number of rows.
+   * @return the new ColumnVector with merged null mask.
+   */
+  public ColumnVector mergeAndSetValidity(BinaryOp mergeOp, ColumnVector... columns) {
```

Comment:
Do we really want it to be a list of columns that we want to merge the validity from? If its not a requirement then I would stick with just a single column to keep it simple. 

Also how about adding a method that takes in a hostbuffer to merge the validity from?

Reply:
It makes sense to merge a list of columns because the underlying cudf call wants a table, doesn't make sense to artificially limit that.

Makes sense to add a method that takes in a hostbuffer, but may be out of the scope and should be done as a separate follow up pr?

Reply:
Only if you feel there's an advantage to doing that. I vaguely remember we can do this in cudf.